examples/cloudflare_delegation/main.tf

```
@@ -0,0 +1,55 @@
+terraform {
+  required_version = ">= 1.6.0"
+
+  backend "s3" {
+    bucket               = "tf-state-911453050078"
+    key                  = "examples/cloudflare.tfstate"
+    workspace_key_prefix = "terraform-aws-kubernetes-platform"
+    dynamodb_table       = "terraform-lock"
+    region               = "eu-central-1"
+  }
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+    cloudflare = {
+      source  = "cloudflare/cloudflare"
+      version = "~> 4.0"
+    }
+  }
+}
+
+data "aws_secretsmanager_secret" "cloudflare" {
+  name = "dai/cloudflare/tamedia/apiToken"
+}
+
+data "aws_secretsmanager_secret_version" "cloudflare" {
+  secret_id = data.aws_secretsmanager_secret.cloudflare.id
```

```suggestion
  secret_id = "dai/cloudflare/tamedia/apiToken"
```

and you can delete the `data "aws_secretsmanager_secret" "cloudflare"` above

---

Why ? I don't get that

---

it's minor and just a shortcut: It's how we do in https://github.com/tx-pts-dai/lacework-settings/blob/c9b22f13a1ababb0d6e653464803fd2b141740a9/account-specific/terraform/main.tf#L37,
you can directly pass the name of the secret in the data.aws_secretsmanager_secret_version, no need to use the data.aws_secrets_manager_secret. just a shortcut to have only 1 data instead of 2 to get the secret value